libs/ui/pagination/helm/src/lib/hlm-numbered-pagination.component.ts

```diff
@@ -53,7 +58,7 @@ import { HlmPaginationDirective } from './hlm-pagination.directive';
 
 					@if (showEdges() && !isLastPageActive()) {
 						<li hlmPaginationItem (click)="goToNext()">
-							<hlm-pagination-next />
+							<hlm-pagination-next class="cursor-pointer" />
```

Comment:
question: What do you think about adding this class conditionally based on `link().trim().length === 0` (disabled) state of `hlm-pagination-next` / `hlm-pagination-previous`?

Reply:
Thats an even better idea! I will see if I can even add it to hlm-pagination-link.